code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -412,7 +438,7 @@
 	. = ..()
 	to_chat(user,"<span class='notice'>You drink the potion and with the viscous liquid, the glass dematerializes.</span>")
 	effect(user)
-	qdel(src)
+	QDEL_NULL(src)
```

Comment:
What's with this change? Nothing will act differently.

Reply:
I know but I've been told fairly often to use this over qdel in the past. This apparently cleans up vars before deletion.